cypress/e2e/albums.cy.js

```diff
@@ -19,116 +19,126 @@
-    cy.unselectMedia([1, 2])
-  })
-
-  it('Download all files from an album', () => {
-    cy.selectMedia([1, 2])
-    cy.downloadSelection()
-    cy.unselectMedia([1, 2])
-  })
-
-  it('Edit an album\'s name', () => {
-    cy.get('[aria-label="Open actions menu"]').click()
-    cy.contains('Edit album details').click()
-    cy.get('form [name="name"]').clear().type("New name")
-    cy.contains('Save').click()
-
-    cy.reload()
-
-    cy.contains('New name')
-
-    cy.get('[aria-label="Open actions menu"]').click()
-    cy.contains('Edit album details').click()
-    cy.get('form [name="name"]').clear().type("albums_test")
-    cy.contains('Save').click()
-  })
-
-  it('Edit an album\'s location', () => {
-    cy.get('[aria-label="Open actions menu"]').click()
-    cy.contains('Edit album details').click()
-    cy.get('form [name="location"]').clear().type("New location")
-    cy.contains('Save').click()
-
-    cy.reload()
-
-    cy.contains('New location')
-
-    cy.get('[aria-label="Open actions menu"]').click()
-    cy.contains('Edit album details').click()
-    cy.get('form [name="location"]').clear()
-    cy.contains('Save').click()
-  })
+	before(function() {
+		cy.createRandomUser()
+			.then((user) => {
+				uploadTestMedia(user)
+				cy.login(user)
+				cy.visit('/apps/photos')
+			})
+	})
+
+	beforeEach(() => {
+		cy.visit(`${Cypress.env('baseUrl')}/index.php/apps/photos/albums`)
+		createAnAlbumFromAlbums('albums_test')
+		addFilesToAlbumFromAlbum('albums_test', [0, 1, 2])
+	})
+
+	afterEach(() => {
+		deleteAnAlbumFromAlbumContent()
+		cy.contains('There is no album yet!').click()
+	})
+
+	it('Add and remove a file to an album from an album', () => {
+		selectMedia([0])
+		removeSelectionFromAlbum()
+	})
+
+	it('Add and remove multiple files to an album from an album', () => {
+		selectMedia([0, 1])
+		removeSelectionFromAlbum()
+	})
+
+	it('Favorite a file from an album', () => {
+		selectMedia([0])
+		favoriteSelection()
+		cy.get('[data-test="media"]').eq(0).find('[aria-label="The file is in the favorites"]')
+		unfavoriteSelection()
+		unselectMedia([0])
+		cy.get('[aria-label="The file is in the favorites"]').should('not.exist')
+	})
+
+	it('Favorite multiple files from an album', () => {
+		selectMedia([1, 2])
+		favoriteSelection()
+		cy.get('[data-test="media"]').eq(1).find('[aria-label="The file is in the favorites"]')
+		cy.get('[data-test="media"]').eq(2).find('[aria-label="The file is in the favorites"]')
+		unfavoriteSelection()
+		unselectMedia([1, 2])
+		cy.get('[aria-label="The file is in the favorites"]').should('not.exist')
+	})
+
+	it('Download a file from an album', () => {
+		selectMedia([0])
+		downloadSelection()
+		unselectMedia([0])
+	})
+
+	it('Download multiple files from an album', () => {
+		selectMedia([1, 2])
+		downloadSelection()
+		unselectMedia([1, 2])
+	})
+
+	it('Download all files from an album', () => {
+		selectMedia([1, 2])
+		downloadSelection()
+		unselectMedia([1, 2])
```

Comment:
Any reason not to make those cypress commands?

Reply:
Mostly typings. Is there a reason to make them cypress commands ? I thought commands were the way to go, but it does not add anything in my experience.